src/BackgroundDimmer/BackgroundDimmer.tsx

```diff
@@ -134,7 +129,7 @@ export default class BackgroundDimmer extends React.PureComponent<BackgroundDimm
     };
 
     return (
-      <div className={cx('background-dimmer', className)} {...omit(props, ['onClickOutside'])}>
+      <div className={cx('background-dimmer', className)} {...omit(props, ['onClickOutside', 'stopScrollPropagation'])}>
```

Comment:
we were not filtering `stopScrollPropagation` (which is not a valid property for `div`).

The tests didn't raise a warning because we were implicitly passing `undefined`, now that we're passing `false` it became evident. I'm not sure whether to consider this as a breaking change (it removes a React warning).

Reply:
@FrancescoCioria you're the `breaking` guy, what do you think?

Reply:
looks like a simple bug/defect so no breaking IMO